Documentation/WritingDocsOfficial/LocalEditing.rst

```diff
@@ -3,111 +3,117 @@
 .. index:: Official documentation; Local editing
 .. _docs-contribute-git-docker:
 
-======================================================
-Workflow #2: "Local editing and rendering with Docker"
-======================================================
+==========================================================
+Workflow #2: Working locally on the documentation with Git
+==========================================================
+
+You can clone the Git repository of a manual to your computer and
+`preview the rendered result locally <rendering-docs>`.
+
+The official manuals of the documentation team can be found in the organization
+`TYPO3-Documentation on GitHub <https://github.com/TYPO3-Documentation>`__.
 
 Using your local machine instead of editing documentation on GitHub has many advantages, it includes
 the freedom to choose which IDE you make your changes in and it also gives you
 the ability to experiment and preview your changes locally before submitting them for approval.
 
-.. rst-class:: bignums-xxl
-
-1. Create a GitHub account:
+..  rst-class:: bignums-xxl
 
-   Visit `Join GitHub <https://github.com/join>`__ and create your account.
+1.  Create a GitHub account:
 
-   Though not mandatory, the general convention in the TYPO3 community is
-   to set your GitHub name (*not* username) as your full name.
+    Visit `Join GitHub <https://github.com/join>`__ and create your account.
 
-2. Find and fork the repository
+    Though not mandatory, the general convention in the TYPO3 community is
+    to set your GitHub name (*not* username) as your full name.
```

Comment:
```suggestion
    to set your GitHub name (*not* username) to your full name.
```

Currently it would suggest your realname to your github name, now its worded properly. However, this is a weird way to say "please act under real names".
Do we really want to recommend a realname Policy?

Reply:
I think we can also remove that